generators/client/templates/angular/src/main/webapp/app/shared/login/_login.component.html

```diff
@@ -23,10 +23,11 @@ <h4 class="modal-title" jhiTranslate="login.title">Sign in</h4>
 </div>
 <div class="modal-body">
     <div class="row">
-        <div class="col-md-4 offset-md-2">
-            <h1 jhiTranslate="login.title">Sign in</h1>
-        </div>
+        <%_ if (enableSocialSignIn) { _%>
+        <div class="col-12">
+        <%_ } else { _%>
         <div class="col-md-8 offset-md-2">
```

Comment:
Th idea of this seems like its was for centering the input boxes.
I think it might be better to use `<div class="col-xs-12">` with a custom class which does `text-align: center` or use the bootstrap option https://v4-alpha.getbootstrap.com/utilities/spacing/#horizontal-centering and get rid of the offset